haskell-language-server.cabal

```diff
@@ -472,7 +473,7 @@ test-suite func-test
   if flag(eval)
     cpp-options: -Deval
 -- formatters
-  if flag(floskell) && (impl(ghc < 9.0.1) || flag(ignore-plugins-ghc-bounds))
+  if flag(floskell) && (impl(ghc < 9.2.1) || flag(ignore-plugins-ghc-bounds))
```

Comment:
It looks like Brittany flags are not reflected here

Reply:
yeah, sadly some of its dependencies still needs `allow-newer`, so it can not be included in the hackage build (there is a comment in cabal-ghc901.project remind us we have to change the condition when is possible)

Reply:
Oh, I missed that one. Thank you for the answer!